static/panes/editor.js

```diff
@@ -298,6 +298,15 @@ Editor.prototype.initCallbacks = function () {
         this.mouseMoveThrottledFunction(e);
     }, this));
 
+    if (window.compilerExplorerOptions.mobileViewer) {
```

Comment:
Maybe add a comment linking to the issue it solves, or just a quick one-liner of why this is here too. This looks great though :)